tdrs-frontend/nginx/local/locations.conf

```diff
@@ -86,6 +87,13 @@ location = / {
     try_files $uri $uri/ /index.html;
 }
 
+proxy_intercept_errors on;
 error_page 500 502 504 /50x.html;
+location = /50x.html {
```

Comment:
TODO: Can be deleted since we don't need interception from Nginx anymore

Reply:
So this should be deleted before merging?

Reply:
it can be deleted now, this is for me not to forget